packages/dex2jar/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pxb1988/dex2jar
+TERMUX_PKG_DESCRIPTION="Tools to work with android .dex and java .class files"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.1.2021.10.31
+TERMUX_PKG_SRCURL=https://github.com/pxb1988/dex2jar/releases/download/v2.2-SNAPSHOT-2021-10-31/dex-tools-2.2-SNAPSHOT-2021-10-31.zip
```

Comment:
Version begins with 2.1-\* but actual tag is 2.2-\*, is that intended?

Reply:
![IMG_20220816_202430](https://user-images.githubusercontent.com/68729523/184911087-9c09f736-1c8b-4fe5-b1ea-6eca8a99702c.jpg)